luigi/contrib/s3.py

```diff
@@ -639,7 +639,7 @@ def __iter__(self):
         self.close()
 
 
-class S3Target(FileSystemTarget):
+class S3Target(FileSystemTarget, LazyFormatMixin):
```

Comment:
Any way to explicitly test this within s3_test's S3Target tests?

Reply:
Anything outside of the FileSystemTarget tests? Originally I didn't have the S3 target updated but the tests failed (correctly) so I had to update S3Target to fix. So it is testing S3